meta-mender-core/recipes-mender/mender-server-certificate/mender-server-certificate.bb

```diff
@@ -0,0 +1,31 @@
+DESCRIPTION = "Mender self-signed server certificate"
+HOMEPAGE = "https://mender.io"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+DEPENDS_append = " ca-certificates"
+RDEPENDS_${PN} = "ca-certificates"
+
+S = "${WORKDIR}"
+localdatadir = "${prefix}/local/share"
+
+inherit allarch
+
+PV = "0.1"
+
+do_install() {
+    if [ ! -f ${WORKDIR}/server.crt ]; then
+        bbfatal "No server server.crt found in SRC_URI"
+    fi
+
+    install -m 0755 -d ${D}${localdatadir}/ca-certificates/mender
+    install -m 0444 ${WORKDIR}/server.crt ${D}${localdatadir}/ca-certificates/mender/server.crt
```

Comment:
In the dunfell case I wrote it should be named `mender-server-certificate.crt`, but I guess I forgot to write it for master. But do you agree it makes sense to install it with this more descriptive name?

Reply:
(I understood it as acceptance criteria for both master and dunfell, but let me elaborate)

My reasoning for shortening the name was that it is in a `mender` directory and has a `.cert` suffix, so `mender/server.cert` is basically the same as `mender-server-certificate.crt`. I don't have a very strong opinion, if you insist I will change it.

Other alternatives could be:
* `mender/server-server.crt`
* `mender/server-self-signed.crt`

Reply:
Yeah ok, that makes sense. Just disregard my comment then.